spacy/pipeline/legacy/entity_linker.py

```diff
@@ -66,10 +66,9 @@ def __init__(
         incl_prior (bool): Whether or not to include prior probabilities from the KB in the model.
         incl_context (bool): Whether or not to include the local context in the model.
         entity_vector_length (int): Size of encoding vectors in the KB.
-        get_candidates (Callable[[KnowledgeBase, Span], Iterable[Candidate]]): Function that
+        get_candidates (Callable[[InMemoryLookupKB, Span], Iterable[Candidate]]): Function that
```

Comment:
I don't think we need to change this type?

Reply:
Was a refactoring mistake. Changed back to `KnowledgeBase`.